.github/workflows/build.yml

```diff
@@ -102,14 +141,22 @@ jobs:
         uses: mxschmitt/action-tmate@v3
         if: env.RUN_TMATE
   test:
+    needs:
+      - diagnostics
+    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
         scenario:
           - default
-    runs-on: ubuntu-latest
+          - systemd_enabled
```

Comment:
You need to remove this line.

Reply:
Should be all better now, 839d752 told me so